tests/tasks/aws/test_s3.py

```diff
@@ -135,3 +135,18 @@ def test_raises_if_bucket_not_eventually_provided(self):
         task = S3List()
         with pytest.raises(ValueError, match="bucket"):
             task.run(prefix="fake/path")
+
+
+class TestS3Move:
+    def test_initialization(self):
+        S3Move()
+
+    def test_initialization_passes_to_task_constructor(self):
+        task = S3Move(name="test", tags=["AWS"])
+        assert task.name == "test"
+        assert task.tags == {"AWS"}
+
+    def test_raises_if_bucket_not_eventually_provided(self):
+        task = S3Move()
+        with pytest.raises(ValueError, match="bucket"):
+            task.run(copy_source="fake/path")
```

Comment:
Since there's some non-trivial logic in this task, I'd recommend using [`moto`](https://github.com/spulec/moto) to write some tests that perform this copy against a mock bucket.